service/src/main/java/bio/terra/profile/app/configuration/LimitsConfiguration.java

```diff
@@ -0,0 +1,8 @@
+package bio.terra.profile.app.configuration;
+
+import java.util.Map;
+import java.util.UUID;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(prefix = "profile.limits")
+public record LimitsConfiguration(Map<UUID, Map<String, String>> subscriptions) {}
```

Comment:
Can you add a brief header comment describing the purpose of this configuration and its intended use?